elements/lisk-bft/src/finality_manager.ts

```diff
@@ -446,6 +431,39 @@ export class FinalityManager extends EventEmitter {
 		return true;
 	}
 
+	public getMaxHeightPrevoted(
+		bftVotingLedgerBuffer: Buffer | undefined,
+	): number {
+		const { ledger } = this._decodeVotingLedger(bftVotingLedgerBuffer);
+		const { preVoted } = this._getChainMaxHeightStatus(ledger);
+
+		return preVoted;
+	}
```

Comment:
This method is used only once in the `BFT`, where its parameter are fetched from the `_chain` module. Since `finalyManager` itself have the access to `_chain`, why not just remove the parameter and use `_chain` directly here.